ext/hiredis_ext/connection.c

```diff
@@ -2,6 +2,63 @@
 #include <errno.h>
 #include "hiredis_ext.h"
 
+#ifdef HAVE_RUBY_FIBER_SCHEDULER_H
+#include "ruby/fiber/scheduler.h"
+#include "ruby/io.h"
+
+// TODO: I copied these from Ruby; are they supposed to be exposed as part of the C extension API?
```

Comment:
How can we get rid of this boilerplate? Seems like a lot of C extensions implementing support for Fiber Scheduler are going to want these utility fns.